index.js

```diff
@@ -162,6 +162,34 @@ ChromeBrowser.prototype = {
 
 ChromeBrowser.$inject = ['baseBrowserDecorator', 'args']
 
+function headlessGetOptions (url, args, parent) {
+  return parent.call(this, url, args).concat(['--headless', '--disable-gpu', '--remote-debugging-port=9222'])
+}
+
+var ChromeHeadlessBrowser = function (baseBrowserDecorator, args) {
+  ChromeBrowser.apply(this, arguments)
+
+  var parentOptions = this._getOptions
+  this._getOptions = function (url) {
+    return headlessGetOptions.call(this, url, args, parentOptions)
+  }
+}
+
+ChromeHeadlessBrowser.prototype = {
+  name: 'ChromeHeadless',
+
+  DEFAULT_CMD: {
+    // Try chromium-browser before chromium to avoid conflict with the legacy
```

Comment:
hmm, yes it is (although it is also out-of-date further up in the file where I copied this block from)